app/views/users/sessions/new.html.erb

```diff
@@ -0,0 +1,30 @@
+<h2>Log in</h2>
+<%= form_with(model: @user, local: true, url: user_session_path, method: :post) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+  
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+<% if flash[:alert] %>
+  <div class="alert alert-danger">
+    <%= flash[:alert] %>
+  </div>
+<% end %>
+
+<%= render "users/shared/links" %>
```

Comment:
- [ ] Please match the design as given on the wireframe 

Design: 
![image](https://user-images.githubusercontent.com/62705947/217943164-1bd64ff7-37f2-416d-926f-d945e700a38e.png)
Yours:
![image](https://user-images.githubusercontent.com/62705947/217943212-1562ba60-52c0-481a-b3d7-3f33230182db.png)

Reply:
- [ ] This is outside the scope of this millstone but I should let you know because the app should not return an error. Currently when I want to create a post with a user it throws me this error after I complete the form 
![image](https://user-images.githubusercontent.com/62705947/217944494-3ae1e2f7-73d7-44de-bb5b-411e47209421.png)